portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,20 +36,33 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+
     
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
+  private final ArrayList<String> comments = new ArrayList<>();
+
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    System.out.println("hi");
     Query query = new Query("Comment").addSort("year", SortDirection.DESCENDING)
                                       .addSort("month", SortDirection.DESCENDING)
                                       .addSort("day", SortDirection.DESCENDING);
 
     PreparedQuery results = datastore.prepare(query);
 
+    int commentAmount = 10;
+    String qString = request.getParameter("comment-amount");
+
+    if (qString != null) {
+      commentAmount = Integer.parseInt(qString);
+    }
```

Comment:
I think we could use a fancy ternary operator here to making things a bit prettier.

`String qString = request.getParameter("comment-ammount");`
`int commentAmount = qString == null ? 10 : Integer.parseInt(qString);`

Reply:
Ooh okay.